packages/vsphere/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.0"
+  changes:
+    - description: Add processor support for vSphere datastore, host and virtualmachine.
```

```suggestion
    - description: Add support for processors in datastore, host and virtualmachine.
```

nit; to match with title.